themes/docs-new/layouts/shortcodes/config_rb_backend_summary.md

```diff
@@ -0,0 +1,17 @@
+The `chef-backend.rb` file is generated using
+`chef-backend-ctl gen-sample-backend-config` and controls most of the
+various feature and configuration flags going into a Chef HA backend
+node. A number of these options control the reliability, stability and
+uptime of the backend PostgreSQL databases, the elastic search index,
```

Comment:
```suggestion
uptime of the backend PostgreSQL databases, the Elasticsearch index,
```

"Elasticsearch" is the name of the product, so this PR should probably have a `s/elastic search/Elasticsearch/g` run on it.